java/common/localstack-test/src/main/java/sleeper/localstack/test/LocalStackTestBase.java

```diff
@@ -78,13 +79,16 @@ public static Set<String> listObjectKeys(String bucketName) {
     }
 
     public static String createFifoQueueGetUrl() {
-        return SQS_CLIENT.createQueue(new CreateQueueRequest()
-                .withQueueName(UUID.randomUUID().toString() + ".fifo")
-                .withAttributes(Map.of("FifoQueue", "true"))).getQueueUrl();
+        return SQS_CLIENT_V2.createQueue(CreateQueueRequest.builder()
+                .queueName(UUID.randomUUID().toString() + ".fifo")
+                .attributes(Map.of(QueueAttributeName.FIFO_QUEUE, "true")).build()).queueUrl();
     }
 
     public static String createSqsQueueGetUrl() {
-        return SQS_CLIENT.createQueue(UUID.randomUUID().toString()).getQueueUrl();
+        return SQS_CLIENT_V2.createQueue(CreateQueueRequest.builder()
+                .queueName(UUID.randomUUID().toString())
+                .build())
+                .queueUrl();
     }
```

Comment:
These changes will conflict with the other branch:
- https://github.com/gchq/sleeper/pull/4909

Please avoid making the same changes on more than one branch.